paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewModel.kt

```diff
@@ -1260,6 +1213,14 @@ internal class CustomerSheetViewModel(
         }
     }
 
+    private suspend fun awaitAttachmentStyle(): CustomerSheetEventReporter.AddPaymentMethodStyle {
```

Comment:
This actually feels like something we should add to `IntegrationMetadata`. Don't we know this up front (during load)?

Reply:
I think we should do that as pre-work, and it'll make the rest of this PR simpler.